index.html

```diff
@@ -4,9 +4,90 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/main.css">
     <title>Project: HTML Forms</title>
 </head>
 <body>
-    
+    <div class="sign-in">
+        <div class="sign-text">
+            <p>Already have an Intuit account?</p>
+            <a href="">I forgot my User ID or Password</a>
+        </div>
+        <button><img src="images/lock.png"> <p>Sign in</p></button>
+    </div>
+    <div class="title-form">
+        <div class="title">
+            <img src="images/intuit.png" alt="">
+        </div>
+        <div class="apps">
+            <div class="diferent-apps">
+                <img class="mint" src="images/mint.png" alt="">
+                    <p>Mint</p>
+            </div>
+            <div class="diferent-apps">
+                <img class="quickbooks" src="images/quickbooks.png" alt="">
+                    <p>quickbooks</p>
+            </div>
+            <div class="diferent-apps">
+                <img class="turbo" src="images/turbo.png" alt="">
+                    <p>TurboTax</p>
+            </div>    
+        </div>
+    </div>
+    <div class="form">
+        <form action="">
+            <div class="form-head">
+                <div class="form-head-title">
+                    <h1>One Account.</h1>
+                    <h1>Everything Intuit.</h1>
+                </div>
+                <div class="form-head-links">
+                    <p>Create an acount for all of intuit's products</p>
+                    <p>including Mint. <a href="">Learn More</a></p>
+                </div>
+            </div>
+            <div class="form-actions">
+                <div class="actions">
+                    <label for="email">Email Addres</label>
+                    <input type="email" id="email" placeholder="E-mail">
+                </div>
+                <div class="actions">
+                    <label for="phone-number">Phone Number</label>
+                    <input type="number" id="phone-number" placeholder="Phone number">
+                </div>
+                <div class="actions">
+                    <label for="password">Password</label>
+                    <input class="password" type="password" id="password" placeholder="Password">
+                </div>
+                <div class="actions">
+                    <label for="confirm-password">Confirm Password</label>
+                    <input type="password" id="confirm-password" placeholder="Password">
+                </div>
+                
+                <button class="create-account"><img src="images/lock.png"> <p>Create Account</p></button>
```

Comment:
Img element should always contain the alt attribute.

Reply:
Solved